modules/material-management/test/multi_keyring.test.ts

```diff
@@ -189,7 +189,21 @@ describe('MultiKeyring: onEncrypt', () => {
     const mkeyring = new MultiKeyringNode({ generator })
     const material = new NodeEncryptionMaterial(suite)
 
-    await expect(mkeyring.onEncrypt(material)).to.rejectedWith(Error)
+    await expect(mkeyring.onEncrypt(material)).to.rejectedWith(Error, 'Generator Keyring has not generated material.')
+  })
+
+  it('Precondition: Only Keyrings explicitly designated as generators can generate material.', async () => {
+    const suite = new NodeAlgorithmSuite(AlgorithmSuiteIdentifier.ALG_AES128_GCM_IV12_TAG16)
+    const child = keyRingFactory({
+      async onEncrypt (material: NodeEncryptionMaterial) {
+        return material
+      },
+      onDecrypt: never
+    })
+
+    const mkeyring = new MultiKeyringNode({  children:[ child ] })
+    const material = new NodeEncryptionMaterial(suite) //.setUnencryptedDataKey(unencryptedDataKey, keyringTrace0)
```

Comment:
Why are we keeping this commented code around?

Reply:
removed